### PR TITLE
Command Tests

### PR DESCRIPTION
**Pull request by ling-ui**
Created utility classes for:
- Task
- Team

Created command tests for:
- Add + Edit Person
- Add + Edit Task
- Add + Edit Team (Missing: Duplicate error test due to existing unfixed bug)
- Find Member
- Find Person
- Task summary
- Clear
- Theme

### CODE REVIEW

File: src/test/java/seedu/address/testutil/PersonUtil.java [hunk 1]
```diff
@@ -38,6 +40,91 @@ public static String getPersonDetails(Person person) {
...
+     */
+    public static String[] convertEditSecondPersonPartialToArgs(Person person) {
+        List<String> argList = new ArrayList<>();
+        argList.add("2");
+        argList.add(FLAG_NAME_STR);
+        argList.add(person.getName().fullName);
+        argList.add(FLAG_PHONE_STR);
+        argList.add(person.getPhone().value);
+        return argList.toArray(new String[0]);
+    }
```
Comment (marcuspang) [inline src/test/java/seedu/address/testutil/PersonUtil.java:112]: Do you think we can abstract it to something like `convertPersonToArgs(Person person, boolean convertName, boolean convertPhone, boolean convertEmail, ...)`

So when we want to convert to args array, we don't have to create a custom one for each partial data?

**Review by marcuspang — COMMENTED** [review]
Looks great! I think the model to args functions can be abstracted better so you don't keep creating new ones

File: src/test/java/seedu/address/testutil/PersonUtil.java (hunk 1 above)
Comment (marcuspang) [reply src/test/java/seedu/address/testutil/PersonUtil.java:112]: Same for the other models

File: src/test/java/seedu/address/logic/commands/EditTaskCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        System.out.println(expectedResult.getFeedbackToUser());
+        assertCommandSuccess(commandToBeTested, model, expectedResult, expectedModel);
+    }
+
+    @Test
+    public void execute_invalidTaskIndexUnfilteredList_failure() {
+        model.getTeam().addMember(TypicalPersons.ALICE);
+        model.getTeam().addTask(TypicalTasks.TASK_1);
+        Task validTask = TypicalTasks.TASK_3;
+        commandLine.parseArgs(TaskUtil.convertEditOutOfBoundsTaskToArgs(validTask));
```
Comment (changjy44) [inline src/test/java/seedu/address/logic/commands/EditTaskCommandTest.java:77]: `convertEditOutOfBoundsTaskToArgs` basically hardcodes the index of the task as 3 and assumes 3 is out of bounds, maybe can make the TaskUtil method to take in an index instead? Then you can pass in TaskList.size() + 1.

Comment (changjy44) [reply src/test/java/seedu/address/logic/commands/EditTaskCommandTest.java:77]: Same for TeamUtil and PersonUtil as well

File: src/test/java/seedu/address/testutil/TeamUtil.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import java.util.List;
+
+import seedu.address.model.team.Team;
+
+/**
+ * A utility class for Team.
+ */
+public class TeamUtil {
+
+    /**
```
Comment (changjy44) [inline src/test/java/seedu/address/testutil/TeamUtil.java:16]: Can you include a description of what each method does?

**Review by changjy44 — APPROVED** [review]
Looks promising, I think can consider refactoring your convertToEditTaskArgs into one single method! Have too many methods not a good idea and can be quite confusing

File: src/test/java/seedu/address/testutil/PersonUtil.java (hunk 1 above)
Comment (marcuspang) [reply src/test/java/seedu/address/testutil/PersonUtil.java]: Disregard, not trivial to accomplish since each command has their own syntax

**Pull request merged**